pkg/module/scaffold/resources/default.yaml

```diff
@@ -0,0 +1,2 @@
+# This is the default Kubernetes CustomResourceDefinition file for the module "{{ .ModuleName}}"
```

Comment:
This file has to contain a default instance of the module Custom Resource, not the Custom Resource Definition.